source/RialtoGStreamerMSEVideoSink.cpp

```diff
@@ -422,6 +429,23 @@ static void rialto_mse_video_sink_set_property(GObject *object, guint propId, co
         }
         break;
     }
+    case PROP_SHOW_VIDEO_WINDOW:
+    {
+        bool showVideoWindow = (g_value_get_boolean(value) == TRUE);
```

Comment:
Should we check for support of show-video-window property in rialto first?

Reply:
I'll add the check